backend/ecommerce/schema.py

```diff
@@ -143,6 +148,46 @@ def mutate(self, info, name, email, phone, password):
         )
 
 
+class LikeReview(graphene.Mutation):
+    id = graphene.String()
+
+    class Arguments:
+        reviewId = graphene.String()
+
+    @login_required
+    def mutate(self, info, reviewId):
+        user = info.context.user
+        print('....................')
+        print(user.id)
+        user = User.objects.get(pk=user.id)
+        print(user)
+        print('....................')
+        review = Review.objects.get(pk=reviewId)
+        like = Like(user=user, review=review)
```

Comment:
you don't need user and review, You can work with id only google you will find easily

Reply:
Thanks! Changing this in all PRs